lib/webassembly.js

```diff
@@ -2,7 +2,7 @@
 // https://developer.mozilla.org/en-US/docs/WebAssembly
 // https://github.com/WebAssembly/design/blob/master/Web.md
 
-type BufferSource = $TypedArray | ArrayBuffer;
+type BufferSource = $TypedArray<number> | $TypedArray<bigint> | ArrayBuffer;
```

Comment:
If you know what would be better, than maintaining `$TypedArray<number> | $TypedArray<bigint>` everywhere, let me know